pfio/NetCDF4_FileFormatter.F90

```diff
@@ -287,11 +287,11 @@ subroutine open(this, file, mode, unusable, comm, info, rc)
       else
          status = nf90_open(file, IOR(omode, NF90_SHARE), this%ncid)
       end if
+      !$omp end critical
       if (status /= nf90_noerr) then
          write(error_unit, fmt='("nf90_open: returned error code (", I0,") opening ", A, " [", A,"]")') &
                status,trim(file),trim(nf90_strerror(status))
       end if
-      !$omp end critical
```

Comment:
Same coment as above. Basically we need the critical section to stay here

Reply:
changed back